test/Dapr.Actors.Test/Runtime/ActorRuntimeTests.cs

```diff
@@ -144,6 +144,136 @@ public async Task TestActorSettings()
 
             element = root.GetProperty("drainRebalancedActors");
             Assert.True(element.GetBoolean());
+
+            try
+            {
+                element = root.GetProperty("reentrancy");
+                Assert.True(false);
+            } catch (System.Collections.Generic.KeyNotFoundException) 
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.trygetproperty?view=net-5.0

Reply:
Noted for the future! With the defaults there I just check those now.